data_steward/tools/snapshot_by_query.py

```diff
@@ -125,7 +147,14 @@ def create_snapshot_dataset(project_id, dataset_id, snapshot_dataset_id):
                         dest='snapshot_dataset_id',
                         help='Name of the new dataset that needs to be created',
                         required=True)
+    parser.add_argument('-ds',
+                        '--data_stage',
```

Comment:
You cannot have multiple characters in short option format, argparse will throw an error. It needs to be a single character. This is because sometimes multiple options can be specified that way (here, `-ds` would mean both `-d` and `-s` options are set)

Reply:
i'll change it to a single character